Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -1123,29 +1099,14 @@ def addTrajectoryWriters(
                 acts.examples.TrackFitterPerformanceWriter(
                     level=acts.logging.INFO,
                     inputParticles="truth_seeds_selected",
-                    inputTrajectories="trajectories",
+                    inputTracks=tracks,
                     inputMeasurementParticlesMap="measurement_particles_map",
                     filePath=str(
                         outputDirRoot / f"performance_track_fitter_{name}.root"
                     ),
                 )
             )
 
-    if outputDirCsv is not None:
-        outputDirCsv = Path(outputDirCsv)
-        if not outputDirCsv.exists():
-            outputDirCsv.mkdir()
-
-        if writeSummary:
-            csvMTJWriter = acts.examples.CsvMultiTrajectoryWriter(
-                level=customLogLevel(),
-                inputTrajectories=trajectories,
-                inputMeasurementParticlesMap="measurement_particles_map",
-                outputDir=str(outputDirCsv),
-                fileName=str(f"tracks_{name}.csv"),
-            )
-            s.addWriter(csvMTJWriter)
-
```

Comment:
Do we need this configuration? If so, we might want to put it back in with the converter in front of it.

Reply:
I think we do not use this in the CI. people might use it offline (@benjaminhuth ?). I wonder if it would make sense to create a CSV track writer for his purpose so we do not mix things up